### DISCUSSION TITLE
new 10.2.0 version with new 9-slice sprites !

### DISCUSSION THREAD

**Discussion by obiot** [melonjs/melonJS#1063] [Announcements]
[melonJS 10.2](https://github.com/melonjs/melonJS/releases/tag/10.2.0) was just released, while it includes its batch of [fixes and small improvements](https://github.com/melonjs/melonJS/compare/10.1.1...10.2.0) (like a new option to improve performance of [me.Text](http://melonjs.github.io/melonJS/docs/me.Text.html) in WebGL), the most major addition is certainly the new [9-slices sprite](http://melonjs.github.io/melonJS/docs/me.NineSliceSprite.html), that will allow for better rendering/drawing of UI elements such as panel or dialog box.

The [UI example](https://melonjs.github.io/examples/examples/UI/) has been updated to use it, and this is so much nicer !

Basically before we had this (with the sprite for the panel being stretched/scaled to fit with the panel size) :
<img width="969" alt="Screenshot_2021-11-12_at_5 33 26_PM" src="https://user-images.githubusercontent.com/4033090/142957441-c3ac5da7-8366-44b3-bdc9-714094a50aa2.png">

And now we have this :
<img width="758" alt="Screenshot_2021-11-12_at_5 34 04_PM" src="https://user-images.githubusercontent.com/4033090/142957435-e470e2c8-1e68-4204-a1a6-05863404e1b6.png">

let us know what you think!
